### DISCUSSION TITLE
Pydantic parsing in Hydra apps! (via hydra-zen)

### DISCUSSION THREAD

**Discussion by rsokl** [pydantic/pydantic#9356] [Projects using Pydantic]
Hydra's CLI grammar and config-composition abilities are very powerful, but Hydra's Its runtime type-checking & value-parsing support are extremely limited. Now, you can add a layer of pydantic parsing and validation to Hydra apps via the [hydra-zen](https://github.com/mit-ll-responsible-ai/hydra-zen) library (a library that makes Hydra much nicer to use).

Check out this how-to guide to grok this at a glance: https://mit-ll-responsible-ai.github.io/hydra-zen/how_to/pydantic_guide.html#pydantic-parsing

**Comment by absudabsu** [comment]
Hi, this is really useful, but I can't figure out how to get `hydra_zen.zen` to only partially instantiate when using `pydantic_parser`. Any ideas?
